src/Sentry.Unity/SentryUnity.cs

```diff
@@ -32,7 +32,9 @@ public static void Init(SentryUnityOptions unitySentryOptions)
             // Uses the game `version` as Release unless the user defined one via the Options
             if (unitySentryOptions.Release == null)
             {
-                unitySentryOptions.Release = $"{Application.productName}@{Application.version}";
+                unitySentryOptions.Release = string.IsNullOrWhiteSpace(Application.productName)
+                    ? $"{Application.version}"
+                    : $"{Application.productName}@{Application.version}";
```

Comment:
`productName` is a native method so it requires P/Invoke to call into:

```
    public static extern string productName { [FreeFunction("GetPlayerSettings().GetProductName"), MethodImpl(MethodImplOptions.InternalCall)] get; }
```

It's better to call it once:

```suggestion
                unitySentryOptions.Release = Application.productName is string productName 
                                         && !string.IsNullOrWhiteSpace(productName)
                ? $"{productName}@{Application.version}"
                : Application.version;
```